playground/AzureSearchEndToEnd/AzureSearch.ApiService/AzureSearch.ApiService.http

```diff
@@ -0,0 +1,6 @@
+@AzureSearch.ApiService_HostAddress = http://localhost:5250
+
+GET {{AzureSearch.ApiService_HostAddress}}/
```

Comment:
```suggestion
@AzureSearch_ApiService_HostAddress = http://localhost:5250

GET {{AzureSearch_ApiService_HostAddress}}/
```

VS doesn't like the `.` .

Reply:
I merged the PR to get it in for the preview4 snap. We can update this in a follow up PR.